Add example of how to use integer values in `postBuild.substitute` (#690)
fluxcd/kustomize-controller: docs/spec/v1beta2/kustomization.md

All the undefined variables in the format `${var}` will be substituted with string empty, unless a default 
is provided e.g. `${var:=default}`.

When using integers as values, for example:

```yaml
apiVersion: kustomize.toolkit.fluxcd.io/v1beta2
kind: Kustomization
metadata:
  name: apps
spec:
  ...
  postBuild:
    substitute:
      AWS_ACCOUNT_ID: "71111111117"
```

> For the integer to be replaced as string, the variable needs to be escaped like so:

Reviewer: ```suggestion
For this integer to be replaced as string, the variable needs to be escaped:
```
